src/main/java/io/pinecone/exceptions/PineconeAlreadyIndexExistsException.java

```diff
@@ -0,0 +1,12 @@
+package io.pinecone.exceptions;
+
+public class PineconeAlreadyIndexExistsException extends PineconeException {
+
+    public PineconeAlreadyIndexExistsException(String message) {
```

Comment:
I was going to mention we might want to swap "Already" and "Index" here as a nit. Given @jhamon's feedback above around making this more general to handle other resource conflicts I think that would be best: https://github.com/pinecone-io/pinecone-java-client/pull/23/files#r1333964728

Reply:
Yeah Index will be unnecessary going forward, so removed it.